### ISSUE TITLE
 "console-command-on-entry" flag don't work

### ISSUE THREAD

**Issue by N4TH4NOT** [isokissa3/WorldGuardExtraFlagsPlugin#53]
Good plugin with great ideas but "console-command-on-entry" flag don't work for me :(
I use Spigot 1.12.2 (lastest version) and WordGuard 6.2.1
Error: &cWorldGuard: An unexpected error occurred! Please see server console.
Error in console : https://pastebin.com/78DfVAfK

Help plz

**Comment by mibby** [comment]
@isokissa3 I'm able to confirm this issue existing as well.

WorldGuard Extra Flags 4.0
```
[20:45:53] [Server thread/INFO]: player issued server command: /region flag SBC console-command-on-entry say
[20:45:53] [Server thread/WARN]: [WorldGuard] WorldGuard encountered an unexpected error
java.lang.NoSuchMethodError: com.sk89q.worldguard.protection.flags.FlagContext.copyWith(Lcom/sk89q/worldedit/extension/platform/Actor;Ljava/lang/String;Ljava/util/Map;)Lcom/sk89q/worldguard/protection/flags/FlagContext;
	at net.goldtreeservers.worldguardextraflags.flags.helpers.CustomSetFlag.parseInput(CustomSetFlag.java:36) ~[?:?]
	at com.sk89q.worldguard.protection.flags.SetFlag.parseInput(SetFlag.java:33) ~[?:?]
	at com.sk89q.worldguard.bukkit.commands.region.RegionCommandsBase.setFlag(RegionCommandsBase.java:395) ~[?:?]
	at com.sk89q.worldguard.bukkit.commands.region.RegionCommands.flag(RegionCommands.java:570) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor319.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_181]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_181]
	at com.sk89q.minecraft.util.commands.CommandsManager.invokeMethod(CommandsManager.java:528) ~[?:?]
	at com.sk89q.minecraft.util.commands.CommandsManager.executeMethod(CommandsManager.java:516) ~[?:?]
	at com.sk89q.minecraft.util.commands.CommandsManager.executeMethod(CommandsManager.java:471) ~[?:?]
	at com.sk89q.minecraft.util.commands.CommandsManager.execute(CommandsManager.java:409) ~[?:?]
	at com.sk89q.worldguard.bukkit.WorldGuardPlugin.onCommand(WorldGuardPlugin.java:285) ~[?:?]
	at com.sk89q.bukkit.util.DynamicPluginCommand.execute(DynamicPluginCommand.java:54) ~[?:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.12.2.jar:git-Paper-1562]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:685) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1476) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1281) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.12.2.jar:git-Paper-1562]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_181]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_181]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:850) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1562]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1562]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
```

**Comment by isokissa3** [comment]
Has been resolved in 07d03b2
